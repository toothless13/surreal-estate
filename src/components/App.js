import React from "react";
import { Routes, Route } from "react-router-dom";
import "../styles/app.css";
import NavBar from "./NavBar";
import Properties from "./Properties";
import AddProperty from "./AddProperty";

const App = () => {
  return (
    <div className="App">
      <NavBar />
      <Routes>
        <Route path="/" element={<Properties />} />
        <Route path="add-property" element={<AddProperty />} />
      </Routes>
    </div>
  );
};

export default App;
