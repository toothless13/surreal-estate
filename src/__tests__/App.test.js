import { render, screen } from '@testing-library/react';
import App from '../components/App.js';

test('renders learn react link', () => {
  render(<App />);
  const headingElement = screen.getByText(/Surreal Estate/i);
  expect(headingElement).toBeInTheDocument();
});
