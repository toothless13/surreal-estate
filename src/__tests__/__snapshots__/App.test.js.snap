// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <img
        alt="surreal estate logo with small castle"
        class="logo"
        src="logo.png"
      />
      <ul
        class="navbar-links"
      >
        <li
          class="navbar-links-item"
        >
          <a
            href="/"
          >
            View Properties
          </a>
        </li>
        <li
          class="navbar-links-item"
        >
          <a
            href="/add-property"
          >
            Add a Property
          </a>
        </li>
      </ul>
    </div>
    <div
      class="property-page-heading"
    >
      Properties Page
    </div>
  </div>
</DocumentFragment>
`;
